scripts/run_docker_build.sh

```diff
@@ -52,7 +52,7 @@ export LANG=en_US.UTF-8
 # These were specific to installing matplotlib. I really want to avoid doing this if possible, but in some cases it
 # is inevitable (without re-implementing a full OS), so I also really want to ensure we can annotate our recipes to
 # state the build dependencies at OS level, too.
-yum install -y libXext libXrender libSM tk libX11-devel
+yum install -y libXext libXrender libSM tk libX11-devel mesa-libGL-devel
```

Comment:
Per your suggestion @ocefpaf. We are adding this to the container in this PR ( https://github.com/pelson/Obvious-CI/pull/63 ). Though we really should work on a better way of doing this that is a bit more transparent to the user and requires only local changes to the recipe itself.